### PR TITLE
Fix windows issues

### PR DESCRIPTION
**Pull request by oifland**
Fixing a bunch of issues

### CODE REVIEW

**Comment by oifland** [comment]
Latest version passes `kitchen verify` correctly. Should be good to go.

File: templates/default/chefdk_bootstrap.psm1.erb [hunk 1]
```diff
@@ -56,8 +56,14 @@ function Set-PSColors {
   }
 }
 
+# Add the tools that come with git (e.g. ssh) to the path
+function Add-Git-Tools-To-Path {
```
Comment (dougireton) [inline templates/default/chefdk_bootstrap.psm1.erb]: Maybe we should use [`windows_path`](https://github.com/chef-cookbooks/windows#windows_path) in the cookbook instead.

**Comment by oifland** [comment]
Re-implemented the path in the git recipe with windows_path. Over to you, @dougireton

**Comment by oifland** [comment]
Tests passing locally (spec, style, verify)

**Pull request merged**